ocean_provider/utils/address.py

```diff
@@ -28,3 +28,9 @@ def get_contract_address(
         for chain_addresses in address_json.values()
         if chain_addresses["chainId"] == chain_id
     )
+
+
+def isAddressMatch(address1: str, address2: str) -> bool:
```

Comment:
I think we already have this somewhere. Either way, you can `return Web3.toChecksumAddress(address1) == Web3.toChecksumAddress(address2)` directly, instead of doing "if True return True"

Reply:
also, please use `snake_case` in Python.